data/docs/migration/migrate-from-honeycomb/data.mdx

```diff
@@ -0,0 +1,263 @@
+                          "trace_id": "%X{trace_id:-}",
+                          "span_id": "%X{span_id:-}"
+                        }
+                    </pattern>
+                </pattern>
+            </providers>
+        </encoder>
+    </appender>
+    <root level="INFO">
+        <appender-ref ref="STDOUT"/>
+    </root>
+</configuration>
+```
+
+**Node.js (Winston):**
+```javascript
+const winston = require('winston');
+const { trace } = require('@opentelemetry/api');
+
+const logger = winston.createLogger({
+  format: winston.format.combine(
+    winston.format.timestamp(),
+    winston.format.json(),
+    winston.format.printf((info) => {
+      const span = trace.getActiveSpan();
+      if (span) {
+        const spanContext = span.spanContext();
+        info.trace_id = spanContext.traceId;
+        info.span_id = spanContext.spanId;
+      }
+      return JSON.stringify(info);
+    })
+  ),
+  transports: [new winston.transports.Console()]
+});
+```
+
+**Python (structlog):**
+```python
+import structlog
+from opentelemetry import trace
+
+def add_trace_info(logger, method_name, event_dict):
+    span = trace.get_current_span()
+    if span:
+        span_context = span.get_span_context()
+        event_dict["trace_id"] = format(span_context.trace_id, "032x")
+        event_dict["span_id"] = format(span_context.span_id, "016x")
+    return event_dict
+
+structlog.configure(
+    processors=[
+        add_trace_info,
+        structlog.processors.JSONRenderer()
+    ]
+)
+```
+
+### Centralized Log Collection
+
+For advanced centralized log collection using OpenTelemetry Collector, see the [SigNoz logs ingestion documentation](https://signoz.io/docs/userguide/logs/).
+
+## Data Retention and Historical Considerations
+
+### Historical Data Migration
+
+**Important**: SigNoz cannot directly import historical data from Honeycomb. Plan your migration strategy accordingly:
+
+| Consideration | Recommendation |
+|---------------|---------------|
+| **Historical Data** | Cannot be migrated - plan for parallel operation during transition |
+| **Retention Overlap** | Run both systems temporarily to maintain historical access |
+| **Compliance Requirements** | Ensure retention policies meet regulatory needs |
+| **Data Export** | Export critical historical data from Honeycomb before migration |
+
+While both platforms use OpenTelemetry for data ingestion, they store data in different formats. Honeycomb uses proprietary event-based storage, while SigNoz stores telemetry data in ClickHouse using schemas optimized for analytical queries. Direct data migration between these systems is not supported.
+
+### Retention Policy Comparison
+
+| Platform | Retention Options | Cost Model |
+|----------|-------------------|------------|
+| **Honeycomb** | Plan-based limits (14-90 days typical) | Per-event pricing |
+| **SigNoz Cloud** | Configurable retention (7-90 days) | Predictable monthly pricing |
```

Comment:
SigNoz cloud doesn't have 7 days of retention option. Please refer our pricing page - https://signoz.io/pricing/#estimate-your-monthly-bill